config.yml

```diff
@@ -0,0 +1,19 @@
+mode:
+  - train
+  - eval
+
+model_types: 
```

Comment:
This should probably be a value not a list. I assume We will Create different configs for each grithm

Reply:
otherwise, if the goal is to turn one off, Just set up a `train: true` and `eval: true`

Reply:
looking at the run file, I think this section is unnecessary. we can just run or not run a model using an enabled: true in each model definition section. if we want to loop things, we can use the hard coded array to find each section.